velox/connectors/hive/HiveConnector.cpp

```diff
@@ -396,7 +400,7 @@ void HiveDataSource::addSplit(std::shared_ptr<ConnectorSplit> split) {
                     std::make_unique<dwio::common::ReadFileInputStream>(
                         fileHandle_->file.get(),
                         dwio::common::MetricsLog::voidLog(),
-                        ioStats_.get()),
+                        ioStatsForStream),
```

Comment:
@oerling The new ioStatsForStream variable is used only in one place, e.g. here. It seems that the original code that passed ioStats_.get() worked the same way. Am I missing something?

Reply:
@oerling Not sure if this comment got lost. Repeating in case it did:

```
The new ioStatsForStream variable is used only in one place, e.g. here. It seems that the original code that passed ioStats_.get() worked the same way. Am I missing something?
```